app/src/main/java/com/nervousfish/nervousfish/modules/constants/Constants.java

```diff
@@ -54,6 +56,23 @@ private Constants(final IServiceLocator serviceLocator) {
         return new ModuleWrapper<>(new Constants(serviceLocator));
     }
 
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getPasswordPath() {
```

Comment:
In principle there shouldn't be an unencrypted database anymore, but I guess we'll discuss that on the next meeting